tests/ui/test_ui.py

```diff
@@ -0,0 +1,30 @@
+"""
+UI tests
+"""
+
+import logging
+
+from ocs_ci.framework.testlib import ManageTest, ui
+from ocs_ci.ocs import constants
+from ocs_ci.utility.openshift_console import OpenshiftConsole
+
+
+logger = logging.getLogger(__name__)
+
+
+@ui
```

Comment:
I think the decorator name should be something like 'console'

Reply:
Used one we already have defined in pytest.ini. See line:

https://github.com/red-hat-storage/ocs-ci/blob/master/pytest.ini#L17